assets/xml/objects/object_b_heart.xml

```diff
@@ -1,8 +1,9 @@
 ﻿<Root>
+    <!-- Unused Heart Piece -->
     <File Name="object_b_heart" Segment="6">
-        <DList Name="object_b_heart_DL_000240" Offset="0x240" />
-        <DList Name="object_b_heart_DL_000348" Offset="0x348" />
-        <Texture Name="object_b_heart_Tex_0003C0" OutName="tex_0003C0" Format="i4" Width="32" Height="32" Offset="0x3C0" />
-        <Texture Name="object_b_heart_Tex_0005C0" OutName="tex_0005C0" Format="i4" Width="16" Height="16" Offset="0x5C0" />
+        <DList Name="gBHeartDL" Offset="0x240" />
+        <DList Name="gBHeartSpriteDL" Offset="0x348" />
+        <Texture Name="gBHeart0Tex" OutName="Heart0" Format="i4" Width="32" Height="32" Offset="0x3C0" />
+        <Texture Name="gBHeart1Tex" OutName="Heart1" Format="i4" Width="16" Height="16" Offset="0x5C0" />
```

Comment:
![Screenshot_2023-10-06_17-17-53](https://github.com/zeldaret/mm/assets/42048411/2c4374c5-e76e-49a9-a2c8-64bf94669f8e)

Can these two be named, or is there too much ambiguity?

Reply:
I do believe that the texture on the left is supposed to be the border of a heart piece, the blue marble-looking border. And the left is just a heart it's a bit complicated, though they are both unused.

Reply:
gBHeart0Tex is used in the gBHeartDL and gBHeart1Tex is used in gBHeartSpriteDL

Reply:
Ah, I'll make sure to give them matching names then

Reply:
Would naming it like that make more sense?


<Root>
    <!-- Unused Heart Container -->
    <File Name="object_b_heart" Segment="6">
        <DList Name="gUnusedHeartContainerDL" Offset="0x240" />
        <DList Name="gUnusedHeart2dDL" Offset="0x348" />
        <Texture Name="gUnusedHeartContainerTex" OutName="unused_heart_container" Format="i4" Width="32" Height="32" Offset="0x3C0" />
        <Texture Name="gUnusedHeart2dTex" OutName="unused_heart_2d" Format="i4" Width="16" Height="16" Offset="0x5C0" />
    </File>
</Root>

Reply:
I think using `Sprite` instead of `2D` has hack-nuss had it makes sense here, that's standard terminology to distinguish the two

Reply:
Bump, not sure if this was seen or not

Reply:
Bump, I agree. Sprite seems better than `2d`.